billthebuilder/meetup-query
# #163279810 Add missing status codes to JSON responses

#### What does this PR do?
* It fixes the json responses to add status codes
#### Description of Task to be completed?
* Status codes are put in every json response
#### How should this be manually tested?
* Clone the repository
* Checkout to the `bg-status-codes-163279810` branch
* Create a virtual environment:`virtualenv venv`
* Activate the virtual environment: `source/venv/bin/activate`
* Install the dependencies: `pip install -r requirements.txt`
* Test the endpoint in postman to see whether proper status codes are being displayed
#### What are the relevant pivotal tracker stories?
* [#163279810](https://www.pivotaltracker.com/story/show/163279810 )
